docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -316,6 +316,50 @@ The configuration above generates the following sidebar:
 	]}
 />
 
+## Custom HTML attributes
+
+Links can also include an `attrs` property to add custom HTML attributes to the link element.
+
+An example use case could be to add a `target="_blank"` attribute to external links to open them in a new tab and also apply a custom `style` attribute to italicize the link label.
```

Comment:
Suggesting a small rewording here inspired by recent guidance from @sarah11918 about introducing code examples:

```suggestion
In the following example, `attrs` is used to add a `target="_blank"` attribute, so that the link opens in a new tab, and to apply a custom `style` attribute to italicize the link label:
```